### PR TITLE
Add yt-dlp Web UI to Automation

### PR DESCRIPTION
**Pull request by arbdevml**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding has working installation instructions.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Review by Teresayluis — APPROVED** [review]

**Review by kokomo123 — CHANGES_REQUESTED** [review]
I suggest you put it after YoutubeDL-Server, as i believe that should be alphabetically. Otherwise, it looks correct to me. Thanks for your contribution!

**Comment by arbdevml** [comment]
> I suggest you put it after YoutubeDL-Server, as i believe that should be alphabetically. Otherwise, it looks correct to me. Thanks for your contribution!

Updated, thank you for your review.

File: README.md [hunk 1]
```diff
@@ -219,9 +219,9 @@ _Related: [Internet of Things (IoT)](#internet-of-things-iot), [Software Develop
 - [ydl_api_ng](https://github.com/Totonyus/ydl_api_ng) - Simple youtube-dl REST API to launch downloads on a distant server. `GPL-3.0` `Python`
 - [YoutubeDL-Material](https://github.com/Tzahi12345/YoutubeDL-Material) - Material Design inspired YouTube downloader, based on youtube-dl. Supports playlists, quality select, search, dark mode and much more, all with a clean and modern design. `MIT` `Nodejs/Docker`
 - [YoutubeDL-Server](https://github.com/nbr23/youtube-dl-server) - Web and REST interface for downloading videos onto a server. `MIT` `Python/Docker`
+- [yt-dlp Web UI](https://github.com/marcopeocchi/yt-dlp-web-ui) - Web GUI for yt-dlp. `MPL-2.0` `TypeScript/Go/Docker`
 - [µTask](https://github.com/ovh/utask) - Automation engine that models and executes business processes declared in yaml. `BSD-3-Clause` `Go/Docker`
 
-
```
Comment (KieranRobson) [inline README.md:224]: Could you replace this removal

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Quick mistake to be corrected

File: README.md (hunk 1 above)
Comment (arbdevml) [reply README.md:224]: thank you for your review.

**Review by kokomo123 — APPROVED** [review]
As soon as the removal is fixed, i think this is all good to go. Thanks for updating and fixing this contribution.

**Comment by arbdevml** [comment]
I apologize for this question, I can not find in GitHub Web UI this option. Could you offer some advice on how to use GitHub Web UI to remove "this removal", please, or create a new pull request?

**Comment by kokomo123** [comment]
it's just a blank line so you may need to delete a space or add a space in the Web UI. Otherwise, i can try to create a new PR that fixes it.

**Review by kokomo123 — APPROVED** [review]
This has been successfully fixed and ready for merging. thanks!

**Review by kokomo123 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -219,6 +219,7 @@ _Related: [Internet of Things (IoT)](#internet-of-things-iot), [Software Develop
 - [ydl_api_ng](https://github.com/Totonyus/ydl_api_ng) - Simple youtube-dl REST API to launch downloads on a distant server. `GPL-3.0` `Python`
 - [YoutubeDL-Material](https://github.com/Tzahi12345/YoutubeDL-Material) - Material Design inspired YouTube downloader, based on youtube-dl. Supports playlists, quality select, search, dark mode and much more, all with a clean and modern design. `MIT` `Nodejs/Docker`
 - [YoutubeDL-Server](https://github.com/nbr23/youtube-dl-server) - Web and REST interface for downloading videos onto a server. `MIT` `Python/Docker`
+- [yt-dlp Web UI](https://github.com/marcopeocchi/yt-dlp-web-ui) - Web GUI for yt-dlp. `MPL-2.0` `TypeScript/Go/Docker`
```
Comment (nodiscc) [inline README.md:222]:
```suggestion
- [yt-dlp Web UI](https://github.com/marcopeocchi/yt-dlp-web-ui) - Web GUI for yt-dlp. `MPL-2.0` `Docker/Go/Nodejs`
```

> The Language tag is the main server-side requirement for the software. Don't include frameworks or specific dialects.

Give the ordering of installation options in the project's README, this makes more sense.

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by arbdevml** [comment]
Thank all of you for your review, is there any issues with it, or should I change something?

**Review by nodiscc — APPROVED** [review]

**Review by kokomo123 — APPROVED** [review]
This fully works out, thanks for fixing all the fixes we have proposed and glad to have this addition to the list!

**Pull request merged**